packages/tailwindcss/src/utilities.test.ts

```diff
@@ -7974,6 +8030,10 @@ test('bg', () => {
       // background-image
       '-bg-none',
       '-bg-gradient-to-br',
+      '-bg-linear-to-br',
+      '-bg-linear-[125deg]', // This could be made valid maybe
+      '-bg-linear-[1.3rad]', // This could be made valid maybe
```

Comment:
I think we should support this personally 👍

Reply:
Updated